apps/desktop/src/routes/[projectId]/+layout.svelte

```diff
@@ -228,11 +230,17 @@
 	{:else if $baseBranch}
 		{#if $mode?.type === 'OpenWorkspace' || $mode?.type === 'Edit'}
 			<div class="view-wrap" role="group" ondragover={(e) => e.preventDefault()}>
-				<Navigation />
-				{#if $showHistoryView}
-					<History onHide={() => ($showHistoryView = false)} />
+				{#if $v3}
+					<Chrome {projectId}>
+						{@render children()}
+					</Chrome>
+				{:else}
+					<Navigation />
+					{#if $showHistoryView}
+						<History onHide={() => ($showHistoryView = false)} />
+					{/if}
```

Comment:
Would this work correctly if we pull it out of the $v3 if condition and put it after? Feels like it'll still render as an overlay, and we could keep it available until we have new history page.

Reply:
Hmm yeah I'll check